src/Room.js

```diff
@@ -154,19 +170,25 @@ class Room extends GameEntity {
       return exits;
     }
 
-    const adjacents = [
+    let adjacents = [
```

Comment:
I'm a sucker for `const`s and refactoring and I wonder if this whole section (lines 173 to 190) could be pulled into a method `getAdjacents` -- would be useful to have this for the end user who may want to check adjacent rooms for their own purposes and stick to the designated diagonal behavior without having to reproduce this code in their bundle.